examples/bridge-app/linux/include/main.h

```diff
@@ -44,3 +44,10 @@ class BridgeCommandDelegate : public NamedPipeCommandDelegate
 public:
     void OnEventCommandReceived(const char * json) override;
 };
+
+// Declare runOnOffRoomAction as an external function that can be called from bridged-actions-stub.cpp
+void runOnOffRoomAction(Room * room, bool actionOn, chip::EndpointId endpointId, uint16_t actionID, uint32_t invokeID,
+                        bool hasInvokeID);
+
+// Declare global variables as extern for use in bridged-actions-stub.cpp
+extern std::vector<Room *> gRooms;
```

Comment:
This duplicates the declaration in that .cpp file..  and neither one should exist, probably.

Reply:
Removed it.